src/app/(main)/dashboard/page.tsx

```diff
@@ -1,12 +1,24 @@
-import React from "react";
-import PreVoyageDashboard from "./components/pre-voyage-dashboard/PreVoyageDashboard";
-import VoyageDashboard from "./components/voyage-dashboard/VoyageDashboard";
+import { redirect } from "next/navigation";
+import { getUser } from "@/utils/getUser";
+import routePaths from "@/utils/routePaths";
 
-function DashboardPage() {
+async function DashboardPage() {
   // TODO: Mocked temporary value
   const isVoyageStarted = true;
```

Comment:
You're going to need to remove the isVoyagestarted check or else it'd never redirect the user to the pre-voyage dashboard.